vsts/generate_javadocs_branch.ps1

```diff
@@ -115,22 +115,22 @@ function CreateJavadocReleaseBranch(
     # Move the generated content to the correct folder. The folder will be different for master and preview branches.
     New-Item -Path ../$FolderName -ItemType Directory
 
-    if($UpdateDepsDocs)
+    if ($UpdateDepsDocs -eq "True")
```

Comment:
This seems to be input from the pipeline as a string and not bool

Reply:
nit and I am ok if we don't to it: maybe do a toLowerString (or whatever the PowerShell equivalence) in case the backend break us by sending us true instead of True.

Reply:
This is sent from the pipeline and I think it will be nice to have if we enter it incorrectly. Will add it. My only laziness is about the wait for pipeline to pass but that is not a good excuse :)

Reply:
Btw I checked and -eq is case insensitive so we are good. you need to do -ceq for a case sensitive comparision on powershell.

Reply:
I think we can even check for both cases with -or ... so if the pipeline ever fixed their issue, we won't break.